.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,7 +1,5 @@
 | - | - |
 | --- | --- |
 | Issue | https://github.com/ripe-tech/ripe-commons-pluginus/issues/XXX |
-| Decisions |   |
-| Animated GIF |   |
-| Dependencies | https://github.com/ripe-tech/ripe-commons-pluginus/pull/XXX |

```

Comment:
Seems like something is still missing: https://raw.githubusercontent.com/ripe-tech/ripe-white/master/.github/PULL_REQUEST_TEMPLATE.md?token=AAXILK4JY2AL4H3VEP2WQZK7RW4E4 (Dependencies section)